updated styles on user server page

KayBeSee opened the pull request:
This is an initial concept for enhancing the UI on BTCPayServer's admin dashboard. If this is a direction we would like to continue to go down, I can continue updating more pages.

Changes:
- change html/body background color from #fff to #f9fafb
- contain content of pages in a block with background #fff and border 1px solid #DFE3E8
- move action button to top right of content
- give active subnav item a bg and side border
- move section title to above subnav

There should probably be a more generalized way of repeating this pattern (classes) and defining it (style guide) but again, that can be addressed. All feedback welcomed and encouraged.

<img width="1440" alt="Screen Shot 2019-05-07 at 8 30 01 AM" src="https://user-images.githubusercontent.com/1429113/57363468-25ad9a00-714f-11e9-8471-0cc282fe6081.png">

pavlenex commented:
I like the concept! Looks quite clean. Wouldn't merge it until it's expanded to other pages.

KayBeSee commented:
Thanks for the feedback and showing me where to edit those files. I'll put in some work over the next few days and expand it out into the whole admin dashboard.

NicolasDorier commented:
@KayBeSee before expanding, let's merge the admin dashboard once @Kukks is happy. I love this design, I can't wait!

NicolasDorier commented:
@pavlenex I prefer we merge little by little. We can release a new version once everything is done.
But basically, once one page is done, the work for the other will be pretty dumb to review in other PR.

britttttk commented:
New Layout 👍

KayBeSee commented:
I removed all the changes from the bootstrap files and moved them into creative.css. 

I'm not sure what to do about the hardcoded colors. I agree that we should preserve the variables to the theming ability, but I'm not sure how I can target those otherwise. Any ideas?

NicolasDorier commented:
Can you fix the build?
```
/source/BTCPayServer/wwwroot/bundles/main-bundle.min.css(4145,32,4145,32): Bundler & Minifier error 0: Expected colon, found '{' [/source/BTCPayServer/BTCPayServer.csproj]
/source/BTCPayServer/wwwroot/bundles/main-bundle.min.css(4173,11,4173,11): Bundler & Minifier error 0: Expected colon, found '{' [/source/BTCPayServer/BTCPayServer.csproj]
/source/BTCPayServer/wwwroot/bundles/main-bundle.min.css : Bundler & Minifier error 0: Unexpected end of file encountered [/source/BTCPayServer/BTCPayServer.csproj]
  	Minified wwwroot/bundles/main-bundle.min.js
  	Minified wwwroot/bundles/jqueryvalidate-bundle.min.js
  	Minified wwwroot/bundles/checkout-bundle.min.css
  	Minified wwwroot/bundles/checkout-bundle.min.js
  	Minified wwwroot/bundles/lightning-node-info-bundle.min.js
  	Minified wwwroot/bundles/cart-bundle.min.js
  	Minified wwwroot/bundles/crowdfund-bundle-1.min.js
  	Bundled wwwroot/bundles/crowdfund-bundle-2.min.js
  	Minified wwwroot/bundles/crowdfund-admin-bundle.min.js
  	Minified wwwroot/bundles/crowdfund-admin-bundle.min.css
  	Minified wwwroot/bundles/crowdfund-bundle.min.css
  	Minified wwwroot/bundles/payment-request-admin-bundle.min.js
  	Minified wwwroot/bundles/payment-request-admin-bundle.min.css
  	Minified wwwroot/bundles/payment-request-bundle-1.min.js
  	Bundled wwwroot/bundles/payment-request-bundle-2.min.js
  	Minified wwwroot/bundles/payment-request-bundle.min.css
  Bundler: Done processing bundleconfig.json

Build FAILED.

/source/BTCPayServer/wwwroot/bundles/main-bundle.min.css(4145,32,4145,32): Bundler & Minifier error 0: Expected colon, found '{' [/source/BTCPayServer/BTCPayServer.csproj]
/source/BTCPayServer/wwwroot/bundles/main-bundle.min.css(4173,11,4173,11): Bundler & Minifier error 0: Expected colon, found '{' [/source/BTCPayServer/BTCPayServer.csproj]
/source/BTCPayServer/wwwroot/bundles/main-bundle.min.css : Bundler & Minifier error 0: Unexpected end of file encountered [/source/BTCPayServer/BTCPayServer.csproj]
    0 Warning(s)
    3 Error(s)

```

NicolasDorier commented on BTCPayServer/wwwroot/main/css/site.css:
```diff
@@ -20,6 +20,70 @@
+    background-color: #d9f7ef;
+    border-left: 5px solid var(--primary);
+    font-weight: 500;
+
```
you forgot `}`

KayBeSee replied:
Ahh, derp. Stupid merge!

Fixed.

KayBeSee commented:
Hmm...not sure why the build is failing...

NicolasDorier commented:
Because you are based on an old stuff that was fixed by @rockstardev not too long ago. You can ignore it.

@Kukks @rockstardev does the CSS changes seems ok?

Kukks commented on BTCPayServer/wwwroot/main/css/site.css:
```diff
@@ -20,6 +20,71 @@
+    font-weight: 500;
+}
+
+.nav-pills .nav-link.active,
```
Are these nav-pills styles repeated in  bootstrap.css?

KayBeSee replied:
[Yes](https://github.com/btcpayserver/btcpayserver/blob/master/BTCPayServer/wwwroot/vendor/bootstrap4/css/bootstrap.css#L4136).

Putting the styles in site.css will overwrite what is in bootstrap.css.

KayBeSee replied:
It might be worth it to namespace it better (`#user .nav-pills .nav-link.active { ... }`) to avoid other active states taking those styles.

NicolasDorier commented:
Can we merge it or still stuff to fix?

Kukks commented:
I think we can merge

Kukks commented:
tests failing though, maybe it needs a rebase

NicolasDorier commented:
@KayBeSee looking forward for the next pages!! :)

NicolasDorier commented:
All other tables look bad though now:


![image](https://user-images.githubusercontent.com/3020646/58478873-04f6c400-8192-11e9-834e-eadb4d9a55e7.png)

![image](https://user-images.githubusercontent.com/3020646/58478915-19d35780-8192-11e9-85ee-f104e541621f.png)

![image](https://user-images.githubusercontent.com/3020646/58478980-3cfe0700-8192-11e9-8c37-6e2a2128049f.png)

@KayBeSee can you fix the other tables?

NicolasDorier commented:
![image](https://user-images.githubusercontent.com/3020646/58479442-43d94980-8193-11e9-96c0-029947cc3d50.png)

NicolasDorier commented:
So I tested it. I think a refactoring should be done where local styles are local to a page instead of in `style.css`. Also, #users stuff should actually be inside a class as we want all array to look the same.

KayBeSee commented:
I was worried those changes might effect other styles. I get another PR opened this evening to prevent changes to other "non-user" tables.

NicolasDorier commented:
I reverted the changes for tables for now.
